Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -1,5 +1,6 @@
 package io.branch.referral;
 
+import static io.branch.referral.BranchError.ERR_BRANCH_ALREADY_INITIALIZED;
```

Comment:
We don't import errors this way for any of the others.

Reply:
Fixed.